src/DynamoCore/DynamoCore.csproj

```diff
@@ -425,5 +429,8 @@ limitations under the License.
     <!-- Generate customization dll for BuiltIn -->
     <GenerateResource UseSourcePath="true" Sources="$(ProjectDir)/BuiltInAndOperators/BuiltInImages.resx" OutputResources="$(ProjectDir)/BuiltInAndOperators/BuiltInImages.resources" References="$(FrameworkAssembliesPath)System.Drawing.dll" />
     <AL TargetType="library" EmbedResources="$(ProjectDir)/BuiltInAndOperators/BuiltInImages.resources" OutputAssembly="$(OutDir)BuiltIn.customization.dll" Version="%(DynamoCoreInfo.Version)" />
+    <!-- Generate customization dll for BuiltIn -->
```

Comment:
why is this being done twice... theres already a customization build step directly above this?

Reply:
@mjkkirschner we had to make a new `images.resx` file just for the `LoopWhile` node which is different from all the other builtin nodes as it is a global designscript method defined in `functionobject.ds`. Without this its icon went missing from the library. @saintentropy pointed this out when he looked at the `IconResourceProvider` code.